runtime-tests/pom.xml

```diff
@@ -218,6 +217,258 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>templating-maven-plugin</artifactId>
```

Comment:
This seems complicated. We should be able to instantiate conditionally. The class will be initialized lazily, so something like this should work:
```java
return Runtime.version().comareTo(Version.parse("21")) >= 0
    ? new SimdInterpreterMachine(instance)
    : new InterpreterMachine(instance);
```
The class won't be loaded on the older JDK version.